src/core/store/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useReducer, useContext } from 'react';
+import { reducerFactory } from '../utils/reducerFactory';
+import { IContextProviderProps, IInitialState } from './interfaces';
+
+import MOVIE from './movie';
+
+export const GlobalContext = createContext({ state: null, dispatch: null });
+
+const initialState: IInitialState = {
+  ...MOVIE.state,
+};
+
+export const actions = {
```

Comment:
Consider renaming this object and the included spread objects from "actions" to "actionTypes".
Reasoning: "actions" are specific types of objects in the context of Redux/ContextAPI. These "actions" objects contain only "type" and "payload" properties.

Reply:
Thanks 😊 You are right, this is more correct